### ISSUE TITLE
Re-organize `Resources` section

### ISSUE THREAD

**Issue by albertkun** [UNopenGIS/7-docs#10]
### Objective
The resources should include the following:

- [ ] Use-cases
- [ ] Data
- [ ] Technical Demos

**Comment by hfu** [comment]
Maybe we can have a brainstorming session on this, and then add information to Resources. Smart Maps Group has a lot of colleagues who knows a lot about Resources. Thanks!

**Comment by albertkun** [comment]
@hfu yes, this would be exciting to brainstorm about! Would you suggest either at the next Documentation team meeting or Smart Maps International meet up?

**Comment by hfu** [comment]
I think we need to keep the size of list good for us to maintain. It should not be too big or too small. Who does what would be important. I think we can brainstorm at the next Documentation team meeting, or the 3rd International Meetup, depending on the priority of this issue in the Documentation project.

**Comment by hfu** [comment]
Regarding Data resources, it would be nice to add data sites of national geospatial information authorities. Some authorities (especially in the north Europe) provide good amount of open geospatial data.

**Comment by albertkun** [comment]
@hfu This is a good idea! Do you think we should put this into a Google Sheet so that it is easy to maintain? I'm happy to think about other open source solutions, like OnlyOffice too! 😄

**Comment by hfu** [comment]
@albertkun Yes, we can start from Google Sheet and then think about making it more open. 

Sometimes I use https://hackmd.io but in some cases documents on hackmd.io is fragile.
